src/Sql/dbo/Stored Procedures/CipherDetails_ReadByUserId.sql

```diff
@@ -1,5 +1,6 @@
 ﻿CREATE PROCEDURE [dbo].[CipherDetails_ReadByUserId]
-    @UserId UNIQUEIDENTIFIER
+    @UserId UNIQUEIDENTIFIER,
+    @Deleted BIT = 0
```

Comment:
I see you are using a default here. Are there touchpoints where we are not explicitly passing it in? I think I'd rather be explicit in this case.